modules/git/command.go

```diff
@@ -11,10 +11,12 @@ import (
 	"io"
 	"os"
 	"os/exec"
+	"runtime"
 	"strings"
 	"time"
 
 	"code.gitea.io/gitea/modules/process"
+	"github.com/mcuadros/go-version"
```

Comment:
github.com/mcuadros/go-version was archived !!!

we should switch to `github.com/hashicorp/go-version`

Reply:
I'll try to create a pr - it shold be a drop in replacement ...